src/com/rakovets/course/practice/string/Task03.java

```diff
@@ -58,6 +70,12 @@ public static void main(String[] args) {
 		//TODO
 		// Код, решающий задачу пишем ниже, при этом используя параметры метода
 		// Для проверки решения необходимо запустить @Test для данного class (в директории test)
-		return null;
+		String strwithspaces = text.replace(";", " ");
+		String[] arrayOfInformation = strwithspaces.split("[\\s]+\\s*");
+		int[] arrayOfSalary = new int[arrayOfInformation.length / 3];
+		for (int i = 0, b =2; i < arrayOfSalary.length; i++, b += 3) {

```

Comment:
Violation of Code Convention
```suggestion
		for (int i = 0, b = 2; i < arrayOfSalary.length; i++, b += 3) {
```